src/Microsoft.AspNet.Mvc.ModelBinding/Binders/BindingSourceModelBinder.cs

```diff
@@ -59,6 +59,10 @@ protected BindingSourceModelBinder([NotNull] BindingSource bindingSource)
         /// <returns>
         /// A <see cref="Task"/> which will complete when model binding has completed.
         /// </returns>
+        /// <remarks>
+        /// Other model binders will never run if this method is called. Return <c>null</c> to skip other model binders
+        /// but allow higher-level handling e.g. falling back to empty prefix.
+        /// </remarks>
```

Comment:
nit: not my behaviour. I just documented something that was `true` before I started. :smiley_cat: